src/journal/journalctl.c

```diff
@@ -1901,6 +1901,24 @@ static int verify(sd_journal *j) {
         return r;
 }
 
+static int watch_run_systemd_journal(int *ret_watch_fd, uint32_t mask) {
```

Comment:
why return the fd via an output param? why not just via the return value?

Reply:
Yeah, I guess there's no reason not to use the return value. Updated.